plugin.go

```diff
@@ -482,3 +496,99 @@ func prepend(prefix, s string) string {
 
 	return s
 }
+
+func GetSlackIdsOfCommitters(p *Plugin) error {
+	if p.Config.CommitterListGitPath == DefaultWorkspace {
+		p.Config.CommitterListGitPath = os.Getenv("DRONE_WORKSPACE")
+	}
+
+	emails, err := getGitEmails(p.Config.CommitterListGitPath)
+	if err != nil {
+		return fmt.Errorf("failed to get git emails: %w", err)
+	}
+	var slackIdsList []string
+	for _, email := range emails {
+		slackId, err := getSlackUserIDByEmail(p.Config.AccessToken, email)
+		if err != nil {
+			//log.Printf("Failed to get Slack ID for email %s: %s\n", email, err.Error())
+			continue
+		}
+
+		slackIdsList = append(slackIdsList, slackId)
+	}
+	slackIdsListStr := strings.Join(slackIdsList, ",")
+	err = WriteEnvToOutputFile("COMMITTER_SLACK_ID_LIST", slackIdsListStr)
```

Comment:
```suggestion
	err = WriteEnvToOutputFile("COMMITTER_SLACK_IDS", slackIdsListStr)
```
also, export the value as JSON array instead of comma separated string